vowpalwabbit/parser/flatbuffer/schema/example.fbs

```diff
@@ -0,0 +1,101 @@
+namespace VW.parsers.flatbuffer;
+
+table Feature {
+  name:string;
```

Comment:
Does this mean that strings are always required? Is it possible to produce a file that only specifies the hashes?

Reply:
Nope, the name is not always required. Can use a file with just hashes as well. In case, the name is provided, the hashes are produced.